tcf_website/static/course/javascript/course_data.js

```diff
@@ -113,10 +129,6 @@ const createChart = grades_data => {
     });
 };
 
-const blankChart = () => {
```

Comment:
This `blankChart` didn't appear to do anything or be used anywhere, so I removed these lines.